undertow/src/main/resources/org/wildfly/extension/undertow/LocalDescriptions.properties

```diff
@@ -374,6 +374,8 @@ undertow.handler.reverse-proxy.host.ssl-context=Reference to the SSLContext to b
 undertow.handler.reverse-proxy.host.security-realm.deprecated=Use the ssl-context attribute to reference a configured SSLContext directly.
 undertow.handler.reverse-proxy.host.enable-http2=If this is true then the proxy will attempt to use HTTP/2 to connect to the backend. If it is not supported it will fall back to HTTP/1.1.
 undertow.handler.reverse-proxy.max-retries=The number of times to attempt to retry a request if it fails. Note that if a request is not considered idempotent then it will only be retried if the proxy can be sure it was not sent to the backend server).
+undertow.handler.reverse-proxy.reuse-x-forwarded-header=Append(reuse existing) remote hop to x-forwarded-* headers, if not, previous value is discarded and remote hop becomes the value.
+undertow.handler.reverse-proxy.rewrite-host-header=Set host header to connection remote end and create x-forwarded-host header.
```

Comment:
"If true set the Host HTTP header to the proxy connection's remote end and create an x-forward-host header."